.github/workflows/otel-infra-helm-test.yml

```
@@ -0,0 +1,24 @@
+name: Otel Infrastructure Collector Helm Install Test
+
+on:
+  pull_request:
+    paths:
+    - 'otel-infrastructure-collector/**'
```

We could specify more precisely in order to 1) not run on doc changes; 2) if we potentially add new platform in the future

```suggestion
    paths:
    - 'otel-infrastructure-collector/k8s-helm/**'
```